.pre-commit-config.yaml

```diff
@@ -1,10 +1,14 @@
+---
```

Comment:
@janosg @tobiasraabe -- can you please double-check that the order of the hooks makes sense for you here and if possible also for other projects? If that is consistent, diffing will be much easier now that the formatting is consistent, too.